.vscode/settings.json

```diff
@@ -61,5 +61,9 @@
   },
   "[handlebars]": {
     "editor.formatOnSave": false
-  }
+  },
+  "cSpell.words": [
+    "beachball",
+    "nuget"
+  ]
```

Comment:
It seems these were added when I clicked "add word to workspace dictionary" while editing in vscode. Seems fine to include?